doc/develop/getting_started/index.rst

```diff
@@ -708,10 +702,35 @@ Here are some next steps for exploring Zephyr:
 * Discover :ref:`project-resources` for getting help from the Zephyr
   community
 
+.. _troubleshooting_installation:
+
+Troubleshooting installation
+****************************
+
+Here are some tips for fixing some issues related to the installation process.
+
+.. _toolchain_zephyr_sdk_update:
+
+Double check Zephyr SDK variables when updating
+===============================================
+
+When updating Zephyr SDK, check whether the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` or :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variables are already set.
+
+* If the variables are not set, the latest compatible version of the Zephyr SDK will be selected by default.
+  Proceed to next step without making any changes.
+
+* If :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` is set, the corresponding toolchain will be selected at build time
+  instead of the Zephyr SDK. Either unset the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` environment variable or change its
+  value to ``zephyr`` to make sure the Zephyr SDK is selected.
```

Comment:
> corresponding toolchain will be selected at build time instead of the Zephyr SDK.

Well, only if the value is different from `zephyr` :wink: 

How about:
```suggestion
If :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` is set, the corresponding toolchain will be selected at build time. Zephyr SDK is identified by the value `zephyr`.
If :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` environment variable is not ``zephyr``, then either unset it or change its value to ``zephyr`` to make sure the Zephyr SDK is selected.
```

Note: this still only refers to the environment variable. In pratice the CMakeCache takes precedence, but I think we can wait for #40389 before cleaning up this.
For now, i'm satisfied with just referring to the envvar to keep things simple for the users.